web/src/main/webapp/ui/dialog.ts

```diff
@@ -327,6 +326,10 @@ export class Dialog implements IHtmlElement {
         this.fields.set(fieldName, {html: input});
     }
 
+    public addText(textString: string): void{
```

Comment:
Please add a comment to this method, similar to the other ones in this file.
Any methods that are used frequently have to be commented.